pkg/apis/pipeline/v1alpha1/pipelinerun_types.go

```diff
@@ -208,3 +211,27 @@ func (pr *PipelineRun) GetServiceAccountName(pipelineTaskName string) string {
 	}
 	return serviceAccountName
 }
+
+// PipelineRunTaskSpec  can be used to configure specific
+// specs for a concrete Task
+type PipelineRunTaskSpec struct {
```

Comment:
I think this is actually this maybe should be either a `PipelineTaskRunSpec` or if we must... a `PipelineRunTaskRunSpec`.